app/controllers/admin/bulk_assignments_controller.rb

```diff
@@ -14,7 +14,7 @@ def create
 
   def enqueue_assignments
     if @bulk_assignment_creation.valid?
-      BulkAssignmentJob.perform_later(create_params)
+      BulkAssignmentJob.perform_later(create_params.to_h)
```

Comment:
`ActionController::Parameters` no longer inherits from `Hash`, so ActiveJob doesn't know how to serialize it.